src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -788,10 +790,11 @@ class ApplicationManagerImpl
   void OnFindNewApplicationsRequest() OVERRIDE;
   void RemoveDevice(
       const connection_handler::DeviceHandle& device_handle) OVERRIDE;
-  bool OnServiceStartedCallback(
+  void OnServiceStartedCallback(
```

Comment:
Done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/5fda0d6bf9f2ae414080184699e49c669b73fbf8.
Yes, the reason of changing the return type is that the returning value is now notified by another method.